internal/backends/collection.go

```diff
@@ -285,6 +286,30 @@ func (cc *collectionContract) CreateIndexes(ctx context.Context, params *CreateI
 	return res, err
 }
 
+// DropIndexesParams represents the parameters of Collection.DropIndexes method.
+type DropIndexesParams struct {
+	Indexes []string       // Index names to drop.
+	Spec    []IndexKeyPair // Single index to drop.
+	DropAll bool           // Drop all indexes except for _id_.
```

Comment:
`DeleteAll` does not accept a filter, `ordered`, etc. Please do the same.

Reply:
Should it take a single parameter `Indexes []IndexInfo`? Handler already hold that info when it called `ListIndexes` before calling `DropIndexes`.